### ISSUE TITLE
Crash occurs when using it with Next.JS 13 (+App Directory)?

### ISSUE THREAD

**Issue by therne** [Darginec05/Yopta-Editor#31]
**Describe the bug**
Hi, First thanks for your amazing work! I've been searching through the editor library exactly like this. 

I'm setting up a project using Next.JS v13 with App Directory feature enabled. However, when I try to use Yopta, it works at the first time but when I try to add a block or change order it crashes with the below error:

https://user-images.githubusercontent.com/1618736/224556042-45eebf44-2aaf-48ae-b2b6-7c93eb78220d.mov

The stacktrace is:

```
Unhandled Runtime Error
TypeError: Cannot read properties of null (reading 'type')

Call Stack
e.isVoid
../node_modules/.pnpm/yopta-editor@1.1.4_yaswuzgb2kjpyjjioxtoccuwlu/node_modules/yopta-editor/dist/index.js (21:20240)
Object.isVoid
../node_modules/.pnpm/slate@0.91.4/node_modules/slate/dist/index.es.js (1246:0)
Jo
../node_modules/.pnpm/yopta-editor@1.1.4_yaswuzgb2kjpyjjioxtoccuwlu/node_modules/yopta-editor/dist/index.js (13:68781)
```

It seems like on `withVoidNodes` function on `Editor`, a `null` is being injected on `element: Element` argument to `Editor.isVoid` handler.

https://github.com/Darginec05/Yopta-Editor/blob/8bc4f3c5a76a0e7a19223fb987e4f6a63c82f855/src/components/Editor/plugins.ts#L9-L17

I cannot give the full code, but it's currently in a vanilla setup like:

```tsx
'use client';

import * as React from 'react';
import { Descendant } from 'slate';
import { YoptaEditor } from 'yopta-editor';
import 'yopta-editor/dist/index.css';

export default function NewGravityPage() {
  const [value, setValue] = useState<Descendant[]>([]);
  return (
    <div ...>
      ...
      <YoptaEditor value={value} onChange={(data) => setValue(data)} />
    </div>
  );
}
```

One suspect is Next.JS v13 App Directory— it changed the default rendering method from client-side to SSR.
I've noticed that after adding yopta-editor, I get multiple "Hydration failed" warnings— which can indicate that it's related to 
 problem between yopta-editor and Next.JS, but I'm not sure of that since I'm a newbie to Next.JS.

Any ideas on how can it be fixed? or can it be solved on v2 you're currently working on?

## Environment Information

**Desktop:**
 - OS: macOS Ventura 13.0.1
 - Browser: Chrome
 - Version: 110.0.5481.177 (arm64)

**Dependency Versions**
- `"slate": "^0.91.4"`
-  `"slate-react": "^0.91.10"`
- `"yopta-editor": "^1.1.4"`
- `"next": "13.2.3"`
- `"react": "18.2.0"`
- `"react-dom": "18.2.0"`

**Comment by Darginec05** [comment]
Hm, strange behaviour, because I provided `<NoSSR />` wrapper for Editor🤔

But thanks for the report! 
I'll check it now

**Comment by Darginec05** [comment]
I checked locally and it works as expected. 
Can you try please with incognito mode?

About warnings. 
I'm getting warnings from browser extenstions, so when I use Yopta-Editor in incognito it works without any warnings.

**Comment by therne** [comment]
@Darginec05 Tried on incognito but no luck 😢 

I've created a sample project which reproduces the issue: could you look at it?
https://github.com/therne/yopta-next-test/

This replicated the same project base from my code:
 * Next.JS v13 + App Directory
 * Same UI Framework (Tailwind, PostCSS, https://github.com/shadcn/ui )

You can fork and run `pnpm dev` to reproduce the issue.
 
The code applied `YoptaEditor` is on this commit: https://github.com/therne/yopta-next-test/commit/550a3f5f82ec3f02059d8f37705a6121aacafb8a

**Comment by Darginec05** [comment]
@therne bug fixed in @1.1.5.
